addons/stock/wizard/stock_picking_return_views.xml

```diff
@@ -33,7 +33,7 @@
                 </group>
                 <footer>
                     <button name="create_returns" string="Return" type="object" class="btn-primary" data-hotkey="q"/>
-                    <button string="Cancel" class="btn-secondary" special="cancel" data-hotkey="z" />
+                    <button string="Discard" class="btn-secondary" special="cancel" data-hotkey="z" />
```

Comment:
Hello @odoo/rd-logistics ,

Do you agree with that change? This is to be consistent with the default button of the "Always edit" feature. Now, the cancel button has `Discard` as label. 🙂 

If not, we can remove that changes. 🙂

Reply:
Seems fine to me, but there are other cases where this probably applies as well (e.g. stock_scrap_form_view2) if we want it to be consistent. Maybe it would make more sense for us to do it ourselves?

Reply:
Hello @ticodoo , yep I agree, in fact, the task related was only for the Services apps but it is not really the case in that PR. 😬 

Hello @roku-odoo , can you remove that change? 🙂

Thanks in advance!

Reply:
> Hello @ticodoo , yep I agree, in fact, the task related was only for the Services apps but it is not really the case in that PR. grimacing
> 
> Hello @roku-odoo , can you remove that change? slightly_smiling_face
> 
> Thanks in advance!

Done.thanks for the review.